pkg/kafka/policy_test.go

```diff
@@ -109,12 +90,18 @@ func (k *kafkaTestSuite) TestProduceRequest(c *C) {
 
 	c.Assert(reqMsg.MatchesRule([]api.PortRuleKafka{
 		{Topic: "foo"},
+	}), Equals, false)
+	c.Assert(reqMsg.MatchesRule([]api.PortRuleKafka{
+		{Topic: "foo"}, {Topic: "bar"},
 	}), Equals, true)
+	c.Assert(reqMsg.MatchesRule([]api.PortRuleKafka{
+		{Topic: "foo"}, {Topic: "baz"},
+	}), Equals, false)
 	c.Assert(reqMsg.MatchesRule([]api.PortRuleKafka{
 		{Topic: "baz"}, {Topic: "foo2"},
 	}), Equals, false)
 	c.Assert(reqMsg.MatchesRule([]api.PortRuleKafka{
-		{Topic: "foo2"}, {Topic: "foo"},
+		{Topic: "bar"}, {Topic: "foo"},
 	}), Equals, true)
```

Comment:
Add a test matching on `{Topic: "foo"}, {Topic: "bar"},{Topic: "baz"}` which must succeed to test the OR behaviour.

Reply:
will do

Reply:
done. tested works as expected